src/OrbitBase/CMakeLists.txt

```diff
@@ -123,28 +123,29 @@ target_sources(OrbitBaseTests PRIVATE
         TaskGroupTest.cpp
         TemporaryFileTest.cpp
         ThreadUtilsTest.cpp
+        TypedefTest.cpp
         UniqueResourceTest.cpp
         WhenAllTest.cpp
         WhenAnyTest.cpp
         WriteStringToFileTest.cpp
 )
 
-if (WIN32)
-target_sources(OrbitBaseTests PRIVATE
-        GetProcAddressTest.cpp)
+if(WIN32)
+        target_sources(OrbitBaseTests PRIVATE
+                GetProcAddressTest.cpp)
 else()
-target_sources(OrbitBaseTests PRIVATE
-        ExecutablePathLinuxTest.cpp
-        ExecuteCommandLinuxTest.cpp
-        GetProcessIdsLinuxTest.cpp)
+        target_sources(OrbitBaseTests PRIVATE
+                ExecutablePathLinuxTest.cpp
+                ExecuteCommandLinuxTest.cpp
+                GetProcessIdsLinuxTest.cpp)
 endif()
 
 # Threadpool test contains some sleeps we couldn't work around - disable them on the CI
 # since they are flaky there (but they work fine on local workstations).
-if (NOT (WIN32 AND "$ENV{QT_QPA_PLATFORM}" STREQUAL "offscreen"))
-target_sources(OrbitBaseTests PRIVATE
-        ThreadPoolTest.cpp
-)
+if(NOT(WIN32 AND "$ENV{QT_QPA_PLATFORM}" STREQUAL "offscreen"))
+        target_sources(OrbitBaseTests PRIVATE
+                ThreadPoolTest.cpp
+        )
```

Comment:
These are unrelated changes and shouldn't be part of the PR

Reply:
Argh. Sorry. Autoformat.